src/components/Process/Process.tsx

```diff
@@ -0,0 +1,176 @@
+import * as React from 'react';
+import { themr, ThemedComponentClass } from 'react-css-themr';
+import { classNames } from '@shopify/react-utilities/styles';
+import { PROCESS } from '../ThemeIdentifiers';
+import * as baseTheme from './Process.scss';
+
+export interface NavState {
+  indx: number;
+  styles: string[];
+}
+
+export interface Step {
+  name: string;
+  component: React.ReactNode;
```

Comment:
@HiirenP user can also pass only the string, its not necessary to pass node always, so add `| string` type as well